R/devmode.R

```diff
@@ -29,39 +29,58 @@ devmode <- function(
 #' @describeIn devmode Determines if Shiny is in Developer Mode. If the `getOption("shiny.devmode")` is set to `TRUE` and not in testing inside `testthat`, then Shiny Developer Mode is enabled.
 #' @section Avoiding direct dependency on shiny:
 #'
-#' The methods explained in this documentation file act independently from the rest of Shiny but are included to provide blue prints for your own packages. If your package already has (or willing to take) a dependency on Shiny, we recommend using the exported methods for consistent behavior.
+#' The methods explained in this help file act independently from the rest of Shiny but are included to provide blue prints for your own packages. If your package already has (or is willing to take) a dependency on \pkg{shiny}, we recommend using the exported Shiny methods for consistent behavior.
```

Comment:
"Note that if you use the exported Shiny methods, calling them will cause the Shiny package to load. This may be undesirable if your code will be used in (for example) R Markdown documents without `runtime: shiny`."